src/_playground/Routes.js

```diff
@@ -39,139 +41,278 @@ import React, { Component } from 'react';
-                                return (
-                                    <NavLink
-                                        activeClassName='frDocs-Nav__item--active'
-                                        className='frDocs-Nav__item'
-                                        key={route.url}
-                                        to={{ pathname: route.url }}>
-                                        {route.name}
-                                    </NavLink>
-                                );
-                            })}
-                        </ul>
+                        <nav>
+                            <ul className='frDocs-Nav'>
+                                <li className='frDocs-Nav__headers'>Getting Started</li>
+                                {routes.sort(sortBy('-section', 'name')).map(route => {
+                                    if (route.section === 'Getting Started') {
+                                        return (
+                                            <li key={route.url}>
+                                                <NavLink
+                                                    activeClassName='frDocs-Nav__item--active'
+                                                    className='frDocs-Nav__item'
+                                                    key={route.url}
+                                                    to={{ pathname: route.url }}>
+                                                    {route.name}
+                                                </NavLink>
+                                            </li>
+                                        );
+                                    }
+                                })}
+                            </ul>
+                        </nav>
+                        <nav>
+                            <ul className='frDocs-Nav'>
+                                <li className='frDocs-Nav__headers'>Components</li>
+                                {routes.sort(sortBy('-section', 'name')).map(route => {
+                                    if (route.section === 'Components') {
+                                        return (
+                                            <li key={route.url}>
+                                                <NavLink
+                                                    activeClassName='frDocs-Nav__item--active'
+                                                    className='frDocs-Nav__item'
+                                                    key={route.url}
+                                                    to={{ pathname: route.url }}>
+                                                    {route.name}
+                                                </NavLink>
+                                            </li>
+                                        );
+                                    }
+                                })}
+                            </ul>
+                        </nav>
```

Comment:
It looks like you're already using the `sort-by` package to sort the routes by section and then by name so I think you could wrap the whole thing in one `.map` loop.  Start with the first section (`<ul>`), render each route within that section, then close the section each time the `section` value changes and then finish with one last closing of the section.

Also, I don't think you want two separate `<nav>` elements.  I think the whole left nav is one `<nav>` element with a separate list (`<ul>`) for each section.